java/src/org/openqa/selenium/chromium/HasPermissions.java

```diff
@@ -0,0 +1,34 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.chromium;
+
+import org.openqa.selenium.Beta;
+
+/**
+ * Used by classes to indicate that they can adjust permissions compatible items.
+ */
+@Beta
+public interface HasPermissions {
```

Comment:
Looking around, it seems like this comes from the [Permissions spec](https://www.w3.org/TR/permissions/) As such, it's not browser-specific, and should probably live in the `remote` package. According to [caniuse](https://caniuse.com/?search=permissions) this is implemented by Edge and Firefox too.

The Safari interface with the same name appears to be different.

Reply:
So, for Firefox, I'm getting: 
> TypeError: 'clipboard-read' (value of 'name' member of PermissionDescriptor) is not a valid value for enumeration PermissionName.
So, I'm not sure it's actually supported the same way? Should this get added to spec (a webdriver section to Permissions I'm assuming) so everyone does the same thing?

Reply:
I'll discuss with Mozilla and change if they do implement the endpoint (didn't look like it in my tests)